ci-operator/step-registry/ibmcloud/provision/bastionhost/ibmcloud-provision-bastionhost-commands.sh

```diff
@@ -115,7 +117,7 @@ EOF
 #dump the info 
 run_command "${IBMCLOUD_CLI} is instance-network-interface-floating-ips ${bastion_name} $nac"
 run_command "${IBMCLOUD_CLI} is sg-rules $sg --vpc ${vpcName}"
-
+run_command "${IBMCLOUD_CLI} is sg $sg"
```

Comment:
This command's output already includes the output from `${IBMCLOUD_CLI} is sg-rules $sg --vpc ${vpcName}` ?

Reply:
removed `run_command "${IBMCLOUD_CLI} is sg-rules $sg --vpc ${vpcName}"`, just keep `run_command "${IBMCLOUD_CLI} is sg $sg"` to check more info (target) of the sg